SparkPost/2web2ui
# SA-1327: Report Builder Metrics Drawer

### What Changed
 - Added Metrics Drawer.

### How To Test
 - go to /reports/summary.
 - enable hibana
 - Open metrics drawer and select some metrics
     - verify that more than 5 can be selected
 - Click `clear metrics`. verify that no metrics are selected.
 - Select some metrics and click 'Apply Metrics`.
 - Verify that the drawer does not close but the summary report has been updated.

### To Do
- [ ] Some metrics differ from mocks. Get UX eyes.
